pkgs/development/haskell-modules/configuration-lts.nix

```diff
@@ -1783,7 +1783,6 @@ self: super: {
   "ceilometer-common" = dontDistribute super."ceilometer-common";
   "cellrenderer-cairo" = dontDistribute super."cellrenderer-cairo";
   "cerberus" = dontDistribute super."cerberus";
-  "cereal" = doDistribute super."cereal_0_5_2_0";
```

Comment:
Hm, that makes it fail again.

I wonder if I should add the same rule for 5.2 or just keep 5.3, since it works.